plugins/modules/zos_copy.py

```diff
@@ -2802,24 +2802,15 @@ def run_module(module, arg_def):
 
             # dest_data_set.type overrides `dest_ds_type` given precedence rules
             if dest_data_set and dest_data_set.get("type"):
-                dest_ds_type = dest_data_set.get("type")
+                dest_ds_type = dest_data_set.get("type").upper()
```

Comment:
This is explicitly calling upper, when the point of this ticket is everything should be lower.  Am I missing something here?

Reply:
That is because data_set.DataSet.MVS_PARTITIONED values are kept in uppercase, as I said in the other comment not every mention to options changed to lowercase. May I ask @rexemin Why were data_set.DataSet frozen sets not updated to lowercase ?

Reply:
I tried to modify as much as possible that didn't require changing every call inside the collection. There are certain utils and system calls that need args to be in uppercase, and to try and hunt them down would block the rest of the PRs for a lot longer, hence why I focused mostly on changing interfaces and just making sure internals still worked correctly.